src/NotFoundPage.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import NotFoundPage from './NotFoundPage';
+
+test('NotFoundPage', () => {
+  const { queryByText } = render((
+    <NotFoundPage />
+  ));
+
+  expect(queryByText('존재하지 않는 페이지 입니다')).not.toBeNull();
```

Comment:
```suggestion
  const { container } = render((
    <NotFoundPage />
  ));

  expect(container).toHaveTextContent('존재하지 않는 페이지 입니다');
```

다른 곳과 마찬가지로 텍스트만 검사한다면 이렇게 하는게 어떨까요?